examples/todo-list/src/__tests__/acceptance/todo-list-todo.acceptance.ts

```diff
@@ -42,7 +42,8 @@ describe('TodoListApplication', () => {
   });
 
   it('creates todo for a todoList', async () => {
-    const todo = givenTodo({todoListId: undefined});
+    const todo = givenTodo();
+    delete todo.todoListId;
```

Comment:
Why is it necessary to delete `todoListId` property, considering that it has been already specified as `todoListId: undefined` when calling `givenTodo`? Can we fix `givenTodo` to honor properties with `undefined` values instead?

Reply:
It does accept it, but this change was just to verify the property wasn't needed at all